routes/api/v1/favorites.js

```diff
@@ -144,7 +144,7 @@ router.get('/', function(req, res, next){
         var i;
         const currentForecasts = []
         for (i = 0; i < location.length; i++) {
-          var url = 'https://api.darksky.net/forecast/' + '80ddbb9666791f550fbdf293adcd6bae/' + location[i].dataValues.latitude + ',' + location[i].dataValues.longitude;
+          var url = 'https://api.darksky.net/forecast/' + ENV['DARKSKY_API_KEY'] + location[i].dataValues.latitude + ',' + location[i].dataValues.longitude;
```

Comment:
Also, you don't need to specify dataValues, I believe it would work with `location[i].latitude`. Found this myself just today :)